src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+.calendar {
+  max-width: 7 * $cell-size + 7 * $cell-gap + 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  margin: 50px auto;
+
+  $days:
+    "mon" 1,
+    "tue" 2,
+    "wed" 3,
+    "thu" 4,
+    "fri" 5,
+    "sat" 6,
+    "sun" 7;
+
+  @each $day, $dayNum in $days {
+    &__start-day--#{$day} {
+      &__cell:nth-child(1) {
+        margin-left: $cell-size*($dayNum - 1) + $cell-gap * $dayNum;
+      }
```

Comment:
It would be better if you delete 19 line and on line 18 add > :nth-child(...).
Also cycle is wrong, it has to move element from left depending on the name of modificator.

Reply:
agree